web/src/test/manual/embed/android-harness/host.js

```diff
@@ -1,6 +1,15 @@
 let x = $('#iframe')[0];
 let ta = $('#ta')[0];
 
+let TEST_KEYBOARD = {
```

Comment:
```suggestion
const TEST_KEYBOARD = {
```

I mean in the scheme of things this is pretty unimportant but heck why not?

Reply:
Oh.  `const` has older support than `let`!  I was worried it was the other way around.  🤦